keps/sig-node/2040-kubelet-cri/README.md

```diff
@@ -113,6 +113,9 @@ list of commits to file.
 - Documentation is updated to reflect beta status. 
 - Update the CI with containerd and CRI-O versions that support the v1 proto.
 - Ensure that the required CRI stats changes are included. See https://github.com/kubernetes/enhancements/issues/2371.
+- Introduce field in the `Image` message to indicate an image should not be garbage collected.
```

Comment:
IIUC Kubelet flag `pod-infra-container-image` is being used to mark pause image and it is not garbage collected, 
Are we going to remove/deprecate the flag? or both will stay together?
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/config/flags.go#L95-L96

Reply:
Eventually I think that flag should be dropped. there is a line below that abstractly describes having the kubelet not think about the pause image. should that be made more concrete?

Reply:
I think to avoid any confusions in future, we can be explicit here that `pod-infra-container-image` will eventually going to be removed (removal is not part of this KEP).

Reply:
added a blurb about it, thanks for the suggestion